### PR TITLE
Update page contents

### PR DESCRIPTION
**Pull request by proton-decay**

### CODE REVIEW

File: library/data:open_food_facts.md [hunk 1]
```diff
@@ -1 +1,43 @@
...
+[in their website](https://world.openfoodfacts.org/data).
+
+# How we use their data
+
+Open Food Facts is probably the best openly available source of data about products (mainly food)
+containing their barcodes (for easy identification), pictures, producers, brands, categories, etc.
+
+We try to match the companies from Open Food Facts to companies from Wikidata,
+but since all we have is just the name, that's not a reliable method.
+Due to amount it the data we cannot match Open Food Facts products to the Wikidata products,
```
Comment (cactus-grande) [inline library/data:open_food_facts.md:42]: Due to the mismatched amount of data..?

Comment (cactus-grande) [inline library/data:open_food_facts.md]: Due to the mismatched amount of data..?

File: library/data:wiki.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+we use [their database dumps](https://www.wikidata.org/wiki/Wikidata:Database_download)
+which are updated periodically and
+can be downloaded from [here](https://dumps.wikimedia.org/wikidatawiki/entities/).
+It's worth to note that even a compressed dump still contains more than 100 GB of data. 
+
+# How we use their data
+
+While Wikidata contains information about many companies and products,
+it's far not enough to infer any sustainability or social responsibility information. 
+We use it mainly as a suplementary source of information to provide some description and pictures,
```
Comment (cactus-grande) [inline library/data:wiki.md]: supplementary* source of information to provide some descriptions*

File: library/data:wiki.md [hunk 3]
```diff
@@ -1 +1,44 @@
...
+and combine those with data from organisations certifing products and companies.
+
+Combination of data from Wikidata and different sources is quite cumbersome.
+Ideally we should combine them only if both the certifier data and Wikidata contain some unique identifier,
```
Comment (cactus-grande) [inline library/data:wiki.md]: unique identifiers*

File: library/data:wiki.md [hunk 4]
```diff
@@ -1 +1,44 @@
...
+e.g. VAT number or internet domain.
+But frequetly they are either not available in Wikidata or certifier does not provide them,
```
Comment (cactus-grande) [inline library/data:wiki.md]: However, they are either not available in Wikidata or are not provided by the certifiers in most cases. Therefore, not many connections can be made in this way.

File: library/data:wiki.md [hunk 5]
```diff
@@ -1 +1,44 @@
...
+so not many connections can be found this way.
+To find more matches we also try looking at company names, but that's not a reliable method,
+because such names are not unique.
+I the future we plan to
```
Comment (cactus-grande) [inline library/data:wiki.md]: In the future, we plan to:

File: library/data:open_food_facts.md (hunk 1 above)
Comment (proton-decay) [reply library/data:open_food_facts.md]: "Due to large amount of data"

**Review by cactus-grande — APPROVED** [review]

**Review by cactus-grande — APPROVED** [review]

**Review by cactus-grande — APPROVED** [review]

**Review by cactus-grande — APPROVED** [review]

**Pull request merged**